src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = new UserService().findByEmail(email);
+        try {
+            if (user.getPassword().equals(password)) {
+                return true;
+            }
+        } catch (NullPointerException e) {
```

Comment:
That's cool you've managed to use exception handling here. However, generally it is a bad practice to use exceptions as control flow as it can lead to confusing and hard-to-maintain code. Instead of catching a `NullPointerException`, it's better to explicitly check if `user` is `null`